diff -Naur logtrail-5.6.3-0.1.23/kibana/logtrail/public/app.js logtrail/kibana/logtrail/public/app.js
--- logtrail-5.6.3-0.1.23/kibana/logtrail/public/app.js	2017-10-22 11:16:52.000000000 +0200
+++ logtrail/kibana/logtrail/public/app.js	2017-11-27 17:27:04.943050259 +0100
@@ -101,7 +101,7 @@
       doSearch(null, 'desc', ['overwrite','reverse'], null);
     } else {
       var timestamp = Date.create($scope.pickedDateTime).getTime();
-      doSearch('gt','asc', ['overwrite','scrollToTop'],timestamp);
+      doSearch('gt','desc', ['overwrite','scrollToTop'],timestamp);
     }
     startTailTimer();
   }
@@ -319,7 +319,7 @@
 
     if ($scope.pickedDateTime != null) {
       var timestamp = Date.create($scope.pickedDateTime).getTime();
-      doSearch('gt','asc', ['overwrite','scrollToTop'],timestamp);
+      doSearch('gt','desc', ['overwrite','scrollToTop'],timestamp);
     } else {
       doSearch(null,'desc', ['overwrite','reverse'],null);
     }
@@ -444,7 +444,7 @@
       var docHeight = angular.element($document).height();
       if (scrollPos >= docHeight) {
         if ($scope.events.length > 0) {
-          doSearch('gte', 'asc', ['append','scrollToView'], lastEventTime - ( selected_index_config.es_index_time_offset_in_seconds * 1000 ));
+          doSearch('gte', 'desc', ['append','scrollToView'], lastEventTime - ( selected_index_config.es_index_time_offset_in_seconds * 1000 ));
         }
         $scope.$apply(updateLiveTailStatus('Live'));
       } else {
@@ -474,7 +474,7 @@
       if (lastEventTime) {
         adjustedLastEventTime = lastEventTime - ( selected_index_config.es_index_time_offset_in_seconds * 1000 );
       }
-      doSearch('gte', 'asc', ['append'], adjustedLastEventTime);
+      doSearch('gte', 'desc', ['append'], adjustedLastEventTime);
     }
   };
 
@@ -560,4 +560,4 @@
       });
     }
   }
-});
\ No newline at end of file
+});
